galaxy/Dockerfile

```diff
@@ -48,220 +49,217 @@ ENV GALAXY_CONFIG_FILE=$GALAXY_CONFIG_DIR/galaxy.yml \
+    --extra-vars flower_db_file=$GALAXY_ROOT_DIR/database/flower.db \
+    --extra-vars flower_user=$GALAXY_USER \
+    --extra-vars flower_group=$GALAXY_USER \
+    --extra-vars flower_venv_user=$GALAXY_USER \
+    --extra-vars flower_venv_group=$GALAXY_USER
+
+RUN mkdir -p /shed_tools $GALAXY_CONFIG_TUS_UPLOAD_STORE \
+    && chown $GALAXY_USER:$GALAXY_USER /shed_tools $GALAXY_CONFIG_TUS_UPLOAD_STORE \
     && ln -s /tool_deps/ $EXPORT_DIR/tool_deps \
     # Configure Galaxy to use the Tool Shed
     && chown $GALAXY_USER:$GALAXY_USER $EXPORT_DIR/tool_deps \
-    && apt update -qq && apt install --no-install-recommends -y ansible \
     && ansible-playbook /ansible/provision.yml \
+    --extra-vars galaxy_server_dir=$GALAXY_ROOT_DIR \
     --extra-vars galaxy_venv_dir=$GALAXY_VIRTUAL_ENV \
-    --extra-vars galaxy_log_dir=$GALAXY_LOGS_DIR \
+    --extra-vars galaxy_logs_dir=$GALAXY_LOGS_DIR \
     --extra-vars galaxy_user_name=$GALAXY_USER \
     --extra-vars galaxy_config_file=$GALAXY_CONFIG_FILE \
     --extra-vars galaxy_config_dir=$GALAXY_CONFIG_DIR \
+    --extra-vars gravity_config_file=$GRAVITY_CONFIG_FILE \
     --extra-vars galaxy_job_conf_path=$GALAXY_CONFIG_JOB_CONFIG_FILE \
     --extra-vars galaxy_job_metrics_conf_path=$GALAXY_CONFIG_JOB_METRICS_CONFIG_FILE \
-    --extra-vars supervisor_manage_slurm="" \
-    --extra-vars galaxy_extras_config_condor=True \
-    --extra-vars galaxy_extras_config_condor_docker=True \
-    --extra-vars galaxy_extras_config_rabbitmq=True \
-    --extra-vars galaxy_extras_config_cvmfs=True \
-    --extra-vars galaxy_extras_config_uwsgi=False \
-    --extra-vars proftpd_db_connection=galaxy@galaxy \
-    --extra-vars proftpd_files_dir=$EXPORT_DIR/ftp \
-    --extra-vars proftpd_use_sftp=True \
-    --extra-vars galaxy_extras_docker_legacy=False \
-    --extra-vars galaxy_minimum_version=19.01 \
+    --extra-vars postgresql_version=$PG_VERSION \
     --extra-vars supervisor_postgres_config_path=$PG_CONF_DIR_DEFAULT/postgresql.conf \
-    --extra-vars supervisor_postgres_autostart=false \
-    --extra-vars nginx_use_remote_header=True \
-    --tags=galaxyextras,cvmfs -c local \
+    --extra-vars redis_venv_dir=$GALAXY_VIRTUAL_ENV \
+    --extra-vars redis_venv_user=$GALAXY_USER \
+    --extra-vars galaxy_user_name=$GALAXY_USER \
+    --extra-vars proftpd_sql_db=galaxy@galaxy \
+    --extra-vars proftpd_sql_user=$GALAXY_USER \
+    --extra-vars proftpd_sql_password=$GALAXY_USER \
+    --extra-vars galaxy_ftp_upload_dir=$EXPORT_DIR/ftp \
+    --extra-vars tus_upload_store_path=$GALAXY_CONFIG_TUS_UPLOAD_STORE \
+    --extra-vars gx_it_proxy_sessions_path=$GALAXY_CONFIG_INTERACTIVETOOLS_MAP \
     && . $GALAXY_VIRTUAL_ENV/bin/activate \
     && pip install WeasyPrint \
     && deactivate \
-    # TODO: no clue why this is needed here again
-    && cd $GALAXY_ROOT && ./scripts/common_startup.sh \
+    # this is needed here again to install any conditional dependencies
```

Comment:
These 2 packages get installed by this step: psycopg2-binary==2.9.9 and drmaa

Reply:
I guess its coming from https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/dependencies/conditional-requirements.txt  so we can simply call this and remove the common_startup for a second time.

Reply:
I will add this

Reply:
We could set [LOAD_GALAXY_CONDITIONAL_DEPENDENCIES](https://github.com/bgruening/docker-galaxy-stable/blob/42e298e95e3c1064b5f13e9467a0f388aed569c5/galaxy/startup.sh#L123) and it would install the necessary packages during startup.

Reply:
Nice, lets do that.